.github/workflows/nightly.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Nightly Rust
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * 3'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+             make build
+             make test
+        env:
+          RUST_VERSION: nightly
+          TAG: nightly
+          IMAGE: 'rustserverless/lambda-rust:nightly'
+  publish:
+    needs: [test]
+    if: github.repository == 'rust-serverless/lambda-rust'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: make publish
+        env:
+          RUST_VERSION: nightly
+          TAG: nightly
+  create_issue:
+    runs-on: ubuntu-latest
+    needs: [publish]
+    if: always() && (needs.publish.result == 'failure')
+    steps:
+      - run: gh issue create --title "Nightly publication failed" --body "Nightly publication failed" --label "bug" -R $GITHUB_REPOSITORY
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Newline is missing

Reply:
Newline added (must have been running out of them that day 😊)